hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestCompactionWithFileGroupReader.scala

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
This test has been updated to have both cases: enabled or disabled partial updates.